language-support/ts/daml-ledger/index.ts

```diff
@@ -124,6 +125,29 @@ function isRecordWith<Field extends string>(field: Field, x: unknown): x is Reco
   return typeof x === 'object' && x !== null && field in x;
 }
 
+/** @internal
+ * exported for testing only
+ */
+export function assert(b: boolean, m: string): void {
+  if (!b) {
+    throw m;
+  }
+}
+
+/** @internal
+ * exported for testing only
+ */
+export function deepCopy<T>(input: T): T {
```

Comment:
Updated to use `lodash` for `_.cloneDeep` instead of custom `deepCopy`, and since `lodash` is there I've also removed `deep-equal` and used `_.isEqual` instead of `deepEqual`.